File: core-java-practice/gcr codebase/programming-elements/PowerCalculate.java
import java.util.Scanner;

public class PowerCalculate {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

    // Ask the user for the base number
        System.out.println("Enter the base number: ");
    double n = sc.nextDouble(); // base number

  // Ask the user for the exponent
          System.out.println("Enter the exponent: ");
        double e = sc.nextDouble(); // exponent

  // Using formula and variables entered from user for power function of the library Math in java
          double p = Math.pow(n, e); // power formula

        // result
             System.out.println("The result of " + n + " raised to the power " + e + " is: " + p);

  sc.close();
    }
}

File: core-c#-practice/gcr codebase/programming-elements/PowerCalculate.cs
using System;
class Power
{
    public static void Main ()
    {
        Console.WriteLine ( " Enter the base value of the number to be calculated : " ) ;
        double a = double.Parse ( Console.ReadLine () ) ; // Taking the base value input from user and converting it to double
        Console.WriteLine ( " Enter the exponent or power value to which the base is to be raised : " ) ;
        double e = double.Parse ( Console.ReadLine () ) ;

        // Now using Math.Pow method to get the final value
        double ans = Math.Pow ( a , e ) ;
        Console.WriteLine ( " The ansewer is " + ans);
        Console.WriteLine ( "Or " + (int)ans + "(approx)"); // approximate answer in int using casting
    }
}
